component documentation/componentDoc.css

```diff
@@ -0,0 +1,190 @@
+:root {
+    --primary-color: white ;
+    --background-color: rgb(29, 27, 27);
+    --secondary-color: #6970e7;
+    --text-color: #737477;
+    --backgroundTwo-color: rgb(34, 32, 32); 
+
+}
+
+body {
+    background-color: var(--background-color);
+    color: var(--primary-color);
+    margin: 0%;
+    padding: 0%;
+}
+
+.navbar {
+    position: fixed;
+    background-color: rgb(34, 32, 32) ;
+    top: 0;
+    padding-left: 0.5rem;
+    padding-right: 0.5rem;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-between;
+    box-sizing: border-box;
+    width: 100%;
+}
+
+.heading__link {
+    text-decoration: none;
+    color: var(--secondary-color);
+     padding: 1rem;
+}
+
+.nav__list {
+    display: flex;
+    flex-wrap: wrap;
+    font-size: 10vw;
+    visibility: visible;
+}
+
+.menu-hamburger {
+    display: none;
+}
+
+.nav__link {
+    text-decoration: none;
+    font-size: large;
+    font-weight: bold;
+    padding: 1rem;
+    color: var(--secondary-color);
+}
+
+.nav__link:hover {
+    background-color: #494ea70e;
+    color: white;
+    text-decoration: underline;
+}
+
+
+body {
+    background-color: var(--background-color);
+    color: var(--primary-color);
+    margin: 0%;
+    padding: 0%;
+}
```

Comment:
you have used body two times, club it together.

Reply:
ok